### DISCUSSION TITLE
Changing Text in an iFrame

### DISCUSSION THREAD

**Discussion by alexhayward** [svrnm/DemoMonkey#179] [Ideas]
I can't seem to get DemoMonkey to work on text that's within an iFrame.  This may be because it's not meant to, but if this could be included as a feature, that'd be great!  If it is able to, can the directions on how to do this be added to the Usage pages please?  Or can you point me to where it is documented if I've missed it!

Thanks for this great tool!

**Comment by svrnm** [comment]
Hey @alexhayward , thanks for raising this discussion. iFrames have been tricky historically, DemoMonkey _sometimes_ works with them, _sometimes_ not, it highly depends on a set of factors (configuration of the iframe, how the iframe is generated, etc.), sometimes DemoMonkey drops some errors to the developer console, so if you can share any insights about the iframe you are trying to manipulate, there might be a solution

**Reply by alexhayward** [reply]
Thanks @svrnm, that makes sense.  I can't see any errors in the developer console at the moment.  Thank you for the suggestion though and the quick response!
